test/integration/build/outputs.tf

```diff
@@ -6,6 +6,10 @@ output "aws_vpc_id" {
   value = aws_vpc.inspec_vpc.0.id
 }
 
+output "aws_vpc_dhcp_inspec_dopt_options_id" {
```

Comment:
Since the update to this output, I have been able to test the integration tests. This seems to have an unexpected side effect of replacing the default `aws_vpc_dhcp_options_id`. I propose that this output be reverted back to`aws_vpc_dhcp_options_id ` and the output located on line 25 removed. Also the `aws_dhcp_options` integration test should be reverted to using the original output name. I have tested this locally and this will pass all integration tests.
When this has been done we should be able to get this merged.